controllers/rosters_controller.js

```diff
@@ -10,29 +10,59 @@ const getUsersBySite = async (req, res) => {
       .from('user_semester_site')
       .where('user_semester_site.user_id', userid);
     const data = await knex
-      .select('user.id', 'name', 'email', 'picture', 'notes')
+      .select(
+        'user.id',
+        'name',
+        'email',
+        'picture',
+        'candy',
+        'hobby',
+        'fact',
+        'notes',
+        'studentSemesters',
+      )
       .from('user_semester_site')
       .rightJoin('user', 'user.id', 'user_semester_site.user_id')
       .where('site_id', siteid[0].site_id)
       .where('role', roleType);
     res.status(200).send(data);
   } catch (error) {
+    console.log(error);
     res.status(500).json({ error });
   }
 };
+
+const getUserSemester = async (req, res) => {
```

Comment:
do we use this?

Reply:
- got rid of this after collapsing semesters column